src/pages/AddPost.js

```diff
@@ -29,44 +31,70 @@ const AddPost = ({ navigation }) => {
     description: "",
     picture: "",
   });
+  const [open, setOpen] = useState(false);
+  const [image, setImage] = useState("");
+  const [taggedUser, setTaggedUser] = useState([]);
   const [showConfirmationModal, setShowConfirmationModal] = useState(false);
-  const { userDetails } = useSelector((state) => state.userDetails);
+  const [users, setUsers] = useState([]);
+  const { userDetails, userList } = useSelector((state) => state.userDetails);
```

Comment:
why { userDetails, userList }  both coming from  state.userDetails? @amitark7 
user list should not be coming from userDetails state